tests/e2e/performance/test_pvc_multiple_creation_deletion_performance.py

```diff
@@ -17,58 +17,72 @@
 @performance
 class TestPVCCreationPerformance(E2ETest):
     """
-    Test to verify PVC creation performance
+    Test to verify PVC creation and deletion performance
     """
 
     pvc_size = "1Gi"
 
     @pytest.fixture()
-    def base_setup(self, request, interface_iterate, storageclass_factory):
+    def base_setup(self, interface_type, storageclass_factory):
         """
         A setup phase for the test
 
         Args:
-            interface_iterate: A fixture to iterate over ceph interfaces
+            interface_type: Interface type
             storageclass_factory: A fixture to create everything needed for a
                 storageclass
         """
-        self.interface = interface_iterate
+        self.interface = interface_type
         self.sc_obj = storageclass_factory(self.interface)
 
-    # @pytest.mark.parametrize(
-    #     argnames=["pvc_size"],
-    #     argvalues=[
-    #         pytest.param(*["1Gi"], marks=pytest.mark.polarion_id("OCS-1225")),
-    #         pytest.param(*["10Gi"], marks=pytest.mark.polarion_id("OCS-2010")),
-    #         pytest.param(*["100Gi"], marks=pytest.mark.polarion_id("OCS-2011")),
-    #         pytest.param(*["1Ti"], marks=pytest.mark.polarion_id("OCS-2008")),
-    #         pytest.param(*["2Ti"], marks=pytest.mark.polarion_id("OCS-2009")),
-    #     ],
-    # )
-
+    @pytest.mark.parametrize(
+        argnames=["interface_type", "bulk_size"],
+        argvalues=[
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, 120],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, 720],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, 120],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, 720],
+                marks=[pytest.mark.performance],
+            ),
+        ],
+    )
     @pytest.mark.usefixtures(base_setup.__name__)
     @polarion_id("OCS-1620")
-    def test_multiple_pvc_creation_measurement_performance(self, teardown_factory):
+    def test_bulk_pvc_creation_deletion_measurement_performance(
+        self, teardown_factory, bulk_size
+    ):
+
         """
-        Measuring PVC creation time of 120 PVCs in 180 seconds
+        Measuring PVC creation and deletion time of bulk_size (120/720) PVCs
 
         Args:
             teardown_factory: A fixture used when we want a new resource that was created during the tests
                                to be removed in the teardown phase.
+            bulk_size: Size of the bulk to be tested
         Returns:
 
         """
-        number_of_pvcs = 120
-        multi_creation_time_limit = 60
-        log.info("Start creating new 120 PVCs")
+        bulk_creation_time_limit = bulk_size / 2
```

Comment:
Do we know what are the numbers we see today for 120 & 720?

Reply:
Yes we do: 

RBD:  

23:17:52 - MainThread - root - INFO - 120 Bulk PVCs creation time is 40.038341 seconds.
00:38:40 - MainThread - root - INFO - 720 Bulk PVCs creation time is 280.852776 seconds.

23:24:44 - MainThread - root - INFO - 120 Bulk PVCs deletion time is 46.113197 seconds.
00:57:15 - MainThread - root - INFO - 720 Bulk PVCs deletion time is 286.213385 seconds.

CephFS: 

01:10:04 - MainThread - root - INFO - 120 Bulk PVCs creation time is 40.424715 seconds.
02:40:01 - MainThread - root - INFO - 720 Bulk PVCs creation time is 282.832763 seconds.

01:18:54 - MainThread - root - INFO - 120 Bulk PVCs deletion time is 46.005821 seconds.
03:00:57 - MainThread - root - INFO - 720 Bulk PVCs deletion time is 286.005817 seconds.

Reply:
Great, thank you, Yuli for sharing the results. 
maybe in a subsequent PR, we should fine-tune the time limit based on these test results. i.e., 50 seconds for 120 PVC delete ops and 300 seconds for 720 PVC delete ops.

Reply:
@keesturam I think that we should address this question to Eran Tamir and implement the limit that he decides. Do you agree?